src/test/java/org/openmrs/contrib/qaframework/automation/AttachmentSteps.java

```diff
@@ -0,0 +1,117 @@
+package org.openmrs.contrib.qaframework.automation;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import org.apache.commons.lang3.StringUtils;
+import org.openmrs.contrib.qaframework.page.AttachmentsPage;
+import org.openmrs.reference.page.HomePage;
+
+import io.cucumber.java.After;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+public class AttachmentSteps extends Steps {
+	
+	private AttachmentsPage attachmentsPage;
+	
+	@After("@selenium")
+	public void destroy() {
+		quit();
+	}
+	
+	@Given("a user logs in, searches a patient and visits Attachment page")
```

Comment:
i don't see your 'is your and visits Attachment page' catered for in the block. and i can see the next step doing that

Reply:
Have written the lines that cater for "visit Attachment Page" in the block